### PR TITLE
Feature/packaging

### PR DESCRIPTION
**Pull request by JeltevanBoheemen**
This PR aims to make a full package out of `sastadev`.

- Make an installable package of *all* the `sastadev` modules, rather than a limited subset.
- Configuration
  - The implementation may seem weird, but I've exhausted options to do it in a different way that ensures both `sasta` and `sastadev` run well.
- Proper test architecture
  - Many of the existing 'tests' are skipped because they do not produce a result, the result is dependent on locally available files, or they produce errors. We will aim to fix this at a later date.

For the individual reviewers:
- @JanOdijk 
  - This has some implications for how you write and use the code, but I have tried to keep these to a minimum. You will need to install the package locally (see README) to work on it. Also, mind the configuration, both setting and reading it. 
  - Please try to follow the installation instructions, and let me know if anything is unclear or does not work. It is probably required that any `sastadev` references in your `PATH` are removed, so only the installed package is read. 

- @bbonf , @Meesch 
  - Can you test this in `auchann`? Clone, install locally (e.g. `pip install /Users/my_name/projects/sastadev' and run the tests.
  - General code review. Most of the changes are moved files.

### CODE REVIEW

**Comment by bbonf** [comment]
This requires minor changes in `mwe-query`, which should be merged when the PR is done: https://github.com/UUDigitalHumanitieslab/mwe-query/pull/9

File: src/sastadev/conf.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#         print('initted')
+
+
+class SastadevConfig:
+    '''Class for keeping track of application configuration'''
+
+    def __init__(self,
+                 ALPINO_HOST: str = 'localhost',
+                 ALPINO_PORT: int = 7001,
+                 LOGGER=logging,
```
Comment (bbonf) [inline src/sastadev/conf.py:12]: Might be better to use `logging.getLogger('sastadev')` instead of the root logger, giving more control to downstream  users

File: src/sastadev/conf.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        self.ALPINO_HOST = ALPINO_HOST
+        self.ALPINO_PORT = ALPINO_PORT
+        self.LOGGER = LOGGER
+        self.SD_DIR = SD_DIR
+        self.DATAROOT = DATAROOT
+        self.PARSE_FUNC = PARSE_FUNC
+        print('Configuration initiated.')
+
+
+settings = SastadevConfig()
```
Comment (bbonf) [inline src/sastadev/conf.py:26]: I would maybe prefer having no settings by default, and getting an exception that says I have to initialize sastadev before using any of its functionality. (Or at least a warning, in case you don't want to introduce breaking changes)
Otherwise it could be hard to tell whether something was imported before being configured.

File: generatemacros.py
Comment (JanOdijk) [inline generatemacros.py:1]: Why was this file deleted. The content is needed. Have you moved it elsewhere?

Comment (JeltevanBoheemen) [reply generatemacros.py:1]: It is copied to `src/sastadev/generatemacros.py`. Somehow Git didn't pick register it as `move` but `delete` of the old file +`create` of the new file.

File: src/sastadev/conf.py (hunk 1 above)
Comment (JeltevanBoheemen) [reply src/sastadev/conf.py:12]: Agree. I'll alter the main script a little bit to also work with that configuration.
